CHANGELOG.md

```diff
@@ -49,6 +49,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Improvements
 
+- [#909](https://github.com/babylonlabs-io/babylon/pull/909) Update tokenfactory upgrade fee params to `ubbn`.
```

Comment:
backport 2 prs ?

Reply:
yeah bumping of the tokenfactory version failed automatic backport so adding it here